stdlib/runtime/runtimeDebug.gr

```diff
@@ -0,0 +1,712 @@
+import WasmI32 from "runtime/unsafe/wasmi32"
```

Comment:
This filename is a little confusing vs the current `runtime/unsafe/debug` file. We can't combine them because `runtime/debug` needs to run in `runtime` mode—maybe we rename this file to `memoryDebug.gr` or something similar?

Reply:
`memoryDebug.gr` sounds good to me.

Reply:
Should I move it outside of the runtime folder as well?